docs/SFORMS_WORKFLOW_DOC.md

```diff
@@ -0,0 +1,22 @@
+# S-Forms publishing workflow documentation
```

Comment:
why so crazy naming convention of the file ? I suggest docs/npm-release-workflow.md